README.md

```diff
@@ -144,6 +144,10 @@ all languages.
 
 ## Release Notes
 
+### Release 2.0.2 (June 3, 2021)
+* Updated to version 2.3.4 of the Amazon Kinesis Client Library for Java.
+* Upgraded multiple dependencies 
```

Comment:
We should make release notes / dependency upgrades separate PRS, and we should link PRS once they're merged in so customers can reference them in release notes.

Also, we need to get these release notes approved by doc writer.

Reply:
Also good to list what dependencies were upgraded specifically (i.e AWS SDK, Jackson, etc.)